content/en/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md

```diff
@@ -89,6 +89,24 @@ The metric `scheduler_pending_pods` comes with a new label `"gated"` to distingu
 has been tried scheduling but claimed as unschedulable, or explicitly marked as not ready for
 scheduling. You can use `scheduler_pending_pods{queue="gated"}` to check the metric result.
 
+## Mutable Pod Scheduling Directives
+
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
+
+Pods with scheduling gates have mutable scheduling directives, with certain constraints. 
+The rules for updating a Pod's scheduling directives are as follows:
+
+1. For `.spec.nodeSelector`, only additions are allowed. If absent, it will be allowed to be set.
+
+2. For `spec.affinity.nodeAffinity`, if nil, then setting anything is allowed.
+
+3. For `.requiredDuringSchedulingIgnoredDuringExecution.NodeSelectorTerms`, the terms are ORed while `nodeSelectorTerms[]`.matchExpressions list and `nodeSelectorTerms[].fieldExpressions` are ANDed. If `NodeSelectorTerms` was empty, it will be allowed to be set. If not empty, then only additions of `NodeSelectorRequirements` to `matchExpressions` or `fieldExpressions` are allowed, and no changes to existing `matchExpressions` and `fieldExpressions` will be allowed.
```

Comment:
```suggestion
3. If `NodeSelectorTerms` was empty, it will be allowed to be set. If not empty, then only additions of `NodeSelectorRequirements` to `matchExpressions` or `fieldExpressions` are allowed, and no changes to existing `matchExpressions` and `fieldExpressions` will be allowed. This is because the terms in `.requiredDuringSchedulingIgnoredDuringExecution.NodeSelectorTerms`, are ORed while the expressions in `nodeSelectorTerms[].matchExpressions` and `nodeSelectorTerms[].fieldExpressions` are ANDed. 
```